src/index.ts

```diff
@@ -1248,12 +1248,18 @@ export function io_ui(output: Digitaljs.Module) {
         }
         if (dev.type == 'Input')
             dev.type = dev.bits == 1 ? 'Button' : 'NumEntry';
-        if (dev.type == 'Output')
-            dev.type = dev.bits == 1 ? 'Lamp' : 'NumDisplay';
+        if (dev.type == 'Output') {
+            if (dev.bits == 1)
+                dev.type = 'Lamp';
+            else if (dev.bits == 8 && dev.label == 'display7')
```

Comment:
I would suggest some naming schema, so that there can be multiple 7-segment outputs in a circuit. E.g. names beginning with `display7_`.

Reply:
Maybe just the beginning with `display7` would be ok? In case of just the one display in a circuit, we wouldn't need to numerate it.

Reply:
Changed it in [this commit](https://github.com/tilk/yosys2digitaljs/pull/12/commits/4d3e1feb9520beae40e6935e5c9999fcbfb22007).